src/data/calculate.ts

```diff
@@ -231,14 +248,34 @@ export const calculatePersonRiskEach = (
   }
 }
 
+const getVaccineMultiplier = (data: CalculatorData): number | null => {
+  if (data.yourVaccineType === '') {
+    return 1
+  }
+  const vaccineMultipliers = Vaccines[data.yourVaccineType].multipliers
+  if (
+    data.yourVaccineDoses >= vaccineMultipliers.length ||
```

Comment:
Took me a while to understand this. I think the name "multipliers" is vague in this case. I'd suggest something that more clearly says "here is a list of multiplier for this vaccine based on the number of doses they've received." I don't have a particular suggestion. Maybe multipliersByDose or something. I'd definitely chang it here. And I'd consider changing it in data.ts and any place that it's referenced.

Reply:
Done.